### ISSUE TITLE
Crucial M.2 NVME 1TB SSD (OS), NO INFO DISPLAYED.

### ISSUE THREAD

**Issue by BCC1** [openhardwaremonitor/openhardwaremonitor#1332]
Crucial M.2 NVMe SSD (OS) info is  displayed only as Generic Hard Disk and shows only Used Space percentage. Secondary HDD (storage), attached USB drive and other hardware info is displayed as expected.

**Comment by Phoenix-** [comment]
Same for Samsung 970 Evo, with samsing Nvme driver(if that matters)
OHM 0.9.5
Windows 10 x64 2004, all updates installed.

**Comment by maxima120** [comment]
Would like to see temperature readings for Gigabyte Aorus NVMe also

**Comment by donpin1** [comment]
Agree, please add NVMe drives and their temperatures. It would be great if you also had a tab that monitored SMART attributes. Nevertheless, this is easily the best program for me that I have come across. Well done.
![Capture](https://user-images.githubusercontent.com/79115442/108458079-bd568b00-72d8-11eb-8f79-44e71e17b614.JPG)
